ui/views_test.py

```diff
@@ -32,6 +32,14 @@ def test_program_liveness(self):
             status_code=200
         )
 
+    def test_unauthenticated_user_redirect(self):
+        """Verify that an unauthenticated user can't visit '/dashboard'"""
+        response = self.client.get('/dashboard/')
```

Comment:
Can you use the `reverse` function here? It looks up the url given the url name, which are specified in `ui/urls.py`